gobblin-runtime/src/main/java/org/apache/gobblin/runtime/AbstractJobLauncher.java

```diff
@@ -644,6 +663,16 @@ public void apply(JobListener jobListener, JobContext jobContext)
     }
   }
 
+  @VisibleForTesting
+  public static long sumWorkUnitsSizes (Collection<WorkUnit> workUnits) {
+    long totalSizeInBytes = 0;
+    for (WorkUnit workUnit : workUnits) {
```

Comment:
This is okay too. can be written with lamba expressions too
`long totalSizeInBytes = workUnits.stream().mapToLong(wu -> Long.parseLong(wu.getProp(ServiceConfigKeys.WORK_UNIT_BYTE_SIZE))).sum();`